cmake/Modules/Findopm-parser.cmake

```diff
@@ -47,6 +47,9 @@ endif ()
 find_package(opm-parser CONFIG HINTS ${_opm_parser_build})
 if (opm-parser_FOUND)
     find_package(ecl REQUIRED)
+    find_package(Boost 1.44.0
+                 COMPONENTS regex filesystem system date_time
```

Comment:
> I guess this should have test as well?

It's a little bit of a grey area.  Ideally you would only need to expose those third-party libraries that are needed to link to (installed versions of) `libopmparser`.  There does however appear to be some kind of interaction between multiple calls to `find_package(Boost)` that renders [`UseDynamicBoost.cmake`](https://github.com/OPM/opm-common/blob/master/cmake/Modules/UseDynamicBoost.cmake) very fragile to ordering.